src/ng2-smart-table/components/cell/cell-types/cell-textarea.component.ts

```diff
@@ -0,0 +1,26 @@
+import { Component, Output, EventEmitter, Input } from '@angular/core';
+
+import { Cell } from '../../../lib/data-set/cell';
+import { DefaultCellType } from './default-cell-type';
+
+@Component({
+  selector: 'cell-type-textarea',
+  template: `
+    <textarea [ngClass]="inputClass"
+              class="form-control"
+              [(ngModel)]="cell.newValue"
+              [name]="cell.getColumn().id"
+              [disabled]="!cell.getColumn().isEditable"
+              (click)="onClick($event)"
```

Comment:
Thanks it indeed helped to clean things up :) but I am still having that problem. The only `*.d.ts` file left is `custom-typings.d.ts`, all the rest that you suggested is gone... Have you got any other idea?

PS. Out of curiosity: is there a ready-to-use npm script (eg. `npm run clean`) that does all the above cleaning at once? Thanks!